### PR TITLE
Nebula65

### PR DESCRIPTION
**Pull request by yiancar**
New keyboard!

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/nebula65/keymaps/default/readme.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+The default keymap for Nebula65. VIA support disabled.
+=========================================================
```
Comment (fauxpark) [inline keyboards/nebula65/keymaps/default/readme.md:2]:
```suggestion
# The default keymap for Nebula65. VIA support disabled.
```

File: keyboards/nebula65/keymaps/via/readme.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+The default keymap for Nebula65. VIA support enabled.
+=========================================================
```
Comment (fauxpark) [inline keyboards/nebula65/keymaps/via/readme.md:2]:
```suggestion
# The default keymap for Nebula65. VIA support enabled.
```

File: keyboards/nebula65/readme.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+Nebula65
+=========
```
Comment (fauxpark) [inline keyboards/nebula65/readme.md:2]:
```suggestion
# Nebula65
```

File: keyboards/nebula65/readme.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+This is a standard fixed layout 65% PCB. It supports VIA, full per-key RGB and underglow RGB.
+
+Keyboard Maintainer: [Yiancar](http://yiancar-designs.com/) and on [GitHub](https://github.com/yiancar)\
+Hardware Supported: A 65% keyboard with STM32F303CC\
+Hardware Availability: https://spaceholdings.net/
```
Comment (fauxpark) [inline keyboards/nebula65/readme.md:8]:
```suggestion
* Keyboard Maintainer: [Yiancar](http://yiancar-designs.com/) and on [GitHub](https://github.com/yiancar)
* Hardware Supported: A 65% keyboard with STM32F303CC
* Hardware Availability: https://spaceholdings.net/
```

**Comment by yiancar** [comment]
Also cleaned the .h file as in nebula 12

File: keyboards/nebula65/readme.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+- Plug In
+- Unplug
+- Release Escape
+
+### Flash
+
+- Unplug
+- Hold Escape
+- Plug In
+- Flash using QMK Toolbox or dfu-util (`make nebula65:<keymap>:dfu-util`)
```
Comment (drashna) [inline keyboards/nebula65/readme.md:32]:
```suggestion
- Flash using QMK Toolbox or dfu-util (`make nebula65:<keymap>:flash`)
```

**Pull request merged**